core/src/main/res/values/strings.xml

```diff
@@ -229,6 +229,8 @@
     <string name="download_type_media">Media file</string>
     <string name="download_type_image">Image</string>
     <string name="download_request_error_dialog_message_prefix">An error occurred when trying to download the file:\u0020</string>
+    <string name="null_value_error_dialog_message_prefix">No URL was found when trying import feed:\u0020</string>
+    <string name="null_value_error_error">No podcast was provided that could be shown:\u0020</string>
```

Comment:
Title and message show basically the same thing. I suggest to just display "Error" in the title instead of adding two strings.

Reply:
I was trying to follow the same "train of thought" and structure of what was already there. I have no idea why the previous person wanted a ":" :)